src/Components/Facility/models.tsx

```diff
@@ -222,6 +223,23 @@ export interface CurrentBed {
   meta: Record<string, any>;
 }
 
+export interface ConsultationBedRequest {
+  bed?: string;
+  consultation: string;
+  limit?: number;
+  offset?: number;
+}
+
+export interface ConsultationBedResponse extends PaginatedResponse<CurrentBed> {
+  is_occuiped?: boolean;
+  created_date?: string;
+  modified_date?: string;
+  name?: string;
+  description?: string;
+  meta?: Record<string, any>;
+  assets_objects?: AssetData[];
+}
```

Comment:
You are adding `created_date`, `modified_date`, `name`, and other attributes at the root of the interface. shouldn't these be properties of AssetBed (item of `results` rather than the `PaginatedResponse` itself?